addons/mail/static/src/models/message_notification_popover_content_view.js

```diff
@@ -0,0 +1,26 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'MessageNotificationPopoverContentView',
+    recordMethods: {
+        /**
+         * @private
+         * @returns {MessageView}
+         */
+        _computeMessageView() {
+            return this.popoverViewOwner.messageViewOwnerAsNotificationContent;
```

Comment:
This is a related. Could be used directly instead of computed

Reply:
changes done